kuma/javascript/src/document.jsx

```diff
@@ -360,19 +359,6 @@ export class DocumentRoute extends Route<DocumentRouteParams, DocumentData> {
                         history.replaceState(url, '', url);
                     }
 
-                    // The tags property was added to the doc API recently
-                    // and the cached JSON blobs may not be updated yet.
-                    // So if we receive data with no tags property, we
-                    // have to add one set to an empty array for backward
-                    // compatibility.
-                    //
-                    // NOTE: as a general matter, because of our S3
-                    // cache we need to do this every time we update
-                    // the doc API to ensure backward compatibility.
-                    if (!documentData.tags) {
-                        documentData.tags = [];
-                    }
```

Comment:
What about removing tags from the Document JSON API?
It's an API and it has a version so perhaps that'll be too big a task. But, who else is using the Document JSON API? Some scrapers maybe?

Reply:
@escattone Thoughts?

Reply:
Let's move this discussion to: https://bugzilla.mozilla.org/show_bug.cgi?id=1563476#c7